teuthology/task/console_log.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+
+from teuthology.orchestra.cluster import Cluster
+
+from . import Task
+
+log = logging.getLogger(__name__)
+
+
+class ConsoleLog(Task):
+    enabled = True
```

Comment:
Nope, it is what defaults the task to being enabled. ``__init__`` disables it conditionally.